webpack/components/Table/components/SortableColumnHeaders.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Th } from '@patternfly/react-table';
+
+const SortableColumnHeaders = ({
+  columnHeaders,
+  pfSortParams,
+  columnsToSortParams,
+}) => (
+  columnHeaders.map(col => (
```

Comment:
There are cases where you leave column headers empty "". 
Your keys in such cases would scream at you no?

Reply:
Scream at me? I hope not!

For empty column headers like checkboxes and action menus, these aren't part of the columnHeaders array. for example
```js
              <Th key="select-all" />
              <SortableColumnHeaders
                columnHeaders={columnHeaders}
                pfSortParams={pfSortParams}
                columnsToSortParams={COLUMNS_TO_SORT_PARAMS}
              />
              <Th key="action-menu" />
```

Reply:
Cool.